src/lib/__tests__/session-cache.test.ts

```
@@ -0,0 +1,251 @@
+  const store: Record<string, string> = {};
+  return {
+    safeLocalStorageGet: jest.fn((key: string) => store[key] || null),
+    safeLocalStorageSet: jest.fn((key: string, value: string) => {
+      store[key] = value;
+    }),
+    safeLocalStorageRemove: jest.fn((key: string) => {
+      delete store[key];
+    }),
+    // Helper to reset the store between tests
+    __resetStore: () => {
+      Object.keys(store).forEach(key => delete store[key]);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)